packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -28,6 +36,47 @@ import { resolve } from 'node:path';
 import { existsSync } from 'node:fs';
 import type { ConfigurationRegistry, IConfigurationNode } from './configuration-registry';
 import type { FilesystemMonitoring } from './filesystem-monitoring';
+import type { PodInfo } from './api/pod-info';
+import { PassThrough } from 'node:stream';
+
+function getContainerStatus(state: V1ContainerState | undefined) {
```

Comment:
```suggestion
function toContainerStatus(state: V1ContainerState | undefined) {
```

it seems this method is like the next one (transform a state to a status) so maybe it should use the same approach with to***()

it seems it's missing the return type as well